libnmstate/appliers/bond.py

```diff
@@ -113,3 +113,17 @@ def get_bond_named_option_value_by_id(option_name, option_id_value):
         with contextlib.suppress(ValueError, IndexError):
             return option_value[int(option_id_value)]
     return option_id_value
+
+
+def iface_state_pre_verify_fix(cur_iface_state):
```

Comment:
I think that the function should be with a dedicated purpose and not with a generic name.
In a plug-able flow, one would just register multiple such handlers.

Reply:
I don't know how this register works. But I will change to dedicated name.